<?xml version="1.0"?>
<launch>

  <arg name="gauss1" default="False"/>
  <arg name="gaden1" default="False"/>

  <arg name="subToR1Pose" default="True"/>
  <arg name="subToR2Pose" default="False"/>
  <arg name="subToR3Pose" default="False"/>
  <arg name="subToR4Pose" default="False"/>
  <arg name="subToR5Pose" default="False"/>

  <include file="$(find mocap_optitrack)/launch/mocap.launch"/>

  <param name = "subToR1Pose" value = "$(arg subToR1Pose)" />
  <param name = "subToR2Pose" value = "$(arg subToR2Pose)" />
  <param name = "subToR3Pose" value = "$(arg subToR3Pose)" />
  <param name = "subToR4Pose" value = "$(arg subToR4Pose)" />
  <param name = "subToR5Pose" value = "$(arg subToR5Pose)" />

  <include file="$(find plume_sims)/launch/gauss1.launch" if="$(eval gauss1)"/>
  <include file="$(find plume_sims)/launch/gaden1.launch" if="$(eval gaden1)"/>

  <node name="plotRobots" pkg="brw_omni_wheel" type="plotGaussianAndRobots.py" output="screen"/>
  <node name="plotRobotsForScreenRecording" pkg="brw_omni_wheel" type="plotGaussianAndRobotsForScreenRecording.py" output="screen"/>

</launch>
